cmd/main.go

```diff
@@ -110,6 +120,37 @@ var runCommand = &cli.Command{
 
 		var pipe pipelines.Pipeline
 
+		// find or download the model
+		ok, err := util.FileSystem.Exists(ctx.Context, modelPath)
+		if err != nil {
+			return err
+		}
+		if !ok {
+			modelsDir := util.PathJoinSafe(hugoDir, "models")
+			modelName := modelPath
+			if strings.Contains(modelName, ":") {
+				modelName = strings.Split(modelPath, ":")[0]
```

Comment:
slightly better if you use modelName here, in case we chain more modifications above

Reply:
I'm actually going to just error if the --model is not a full model path and contains ":". This ":" thing is used for model/dataset filters, not super important to support it at the moment.